src/Cronofy.php

```diff
@@ -1140,6 +1140,19 @@ public function executeBatch(Batch $batch): BatchResult
         return $result;
     }
 
+    public function hmacMatch($params, $body)
```

Comment:
I think the name might want to be more verb-like, what do you think?
Also tempted to change $params to be more descripvtive as this isn't making an API call, so it's less likely to be extended. Maybe it can just be `hmac_header`, or take the response and we read the header off that?

```suggestion
    public function verifyHMAC($params, $body)
```

Reply:
happy to modify the function to verifyHMAC :) Regarding the $params, I have used that name to be consistent with other functions, but also happy to change it too.